Geometry/Records/src/ES_MuonDDDConstants.cc

```diff
@@ -0,0 +1,4 @@
+#include "Geometry/MuonNumbering/interface/MuonDDDConstants.h"
```

Comment:
Well Geometry/MuonNumbering/interface/MuonDDDConstants.h is there in the IB  on top of which this branch was made - it is made on top of  CMSSW_7_6_X_2015-06-30-2300. I did not touch anything about Muon here. I removed the merge conflict and made a new tag.
________________________________
From: David Lange [notifications@github.com]
Sent: 01 July 2015 22:27
To: cms-sw/cmssw
Cc: Sunanda Banerjee
Subject: Re: [cmssw] bsunanda:Run2 hcx17 (#9994)


In Geometry/Records/src/ES_MuonDDDConstants.cc<https://github.com/cms-sw/cmssw/pull/9994#discussion_r33720478>:

> @@ -0,0 +1,4 @@
> +#include "Geometry/MuonNumbering/interface/MuonDDDConstants.h"


https://github.com/cms-sw/cmssw/blob/68b4ccfc9d6f7b0ae0980969622686957e61fc9a/Geometry/MuonNumbering/interface/MuonDDDConstants.h ?
…
On Jul 1, 2015, at 10:21 PM, Ianna Osborne <notifications@github.com> wrote: In Geometry/Records/src/ES_MuonDDDConstants.cc: > @@ -0,0 +1,4 @@ > +#include "Geometry/MuonNumbering/interface/MuonDDDConstants.h" @cmsbuild<https://github.com/cmsbuild> - it is very strange that this file is not in IB. Any idea what's wrong? — Reply to this email directly or view it on GitHub.

—
Reply to this email directly or view it on GitHub<https://github.com/cms-sw/cmssw/pull/9994/files#r33720478>.